### DISCUSSION TITLE
Which song to add next?

### DISCUSSION THREAD

**Discussion by ClosedSourcerer** [ClosedSourcerer/ExpertGroupGitHubTraining#4] [Polls]
Which song to add next?

**Comment by pdequick** [comment]
Frère Jacques would be great signal for French hub. Also a great song

**Reply by LukasSchmetz** [reply]
Let´s make our own DCM song

**Comment by lanfrans** [comment]
I would prefer "O mia bela madunina" ....

**Comment by ClosedSourcerer** [comment]
@jschu1
